www/src/components/guidelines/color/modal.js

```diff
@@ -178,7 +205,13 @@ const modalContent = (palette, color) => {
               {c.contrast.colorOnWhite.toFixed(2)} /{` `}
               {getA11yLabel(c.a11y)}
             </Text>
-            <Box width="100%" flexShrink={0} css={{ "& p": { color: c.hex } }}>
+            <Box
+              sx={{
+                width: `100%`,
+                flexShrink: 0,
+              }}
+              css={{ "& p": { color: c.hex } }}
```

Comment:
I think these should be fine to drop in the `sx` object — nesting should work the same as it does for the `css` prop.